src/plugins/validation/validator.js

```diff
@@ -1,6 +1,8 @@
 import PluginComponent from '../plugin.component';
 import * as validationService from '@grid/core/validation/validation.service';
+import {isString} from '@grid/core/utility';
 
+const isArray = angular.isArray;
```

Comment:
Why don't you use ```isArray``` from utility?

Reply:
This is a good question about isArray. Firtly, I used Array.isArray ES6 method, but our interpreter cursed on me and showed me that I should use angular.isArray instead. So, I used angular.isArray and it is OK for now. But maybe in the future This code will be moved to the core. So this is reasonable to use utility isArray method there.

Reply:
@grid/core/utility contains isArray

Reply:
I know